README.md

```diff
@@ -44,3 +44,15 @@ Linters order above is the preffered way to run and fix them one by one.
     or `set ...` for Windows.
 3. Type `flask run` command in terminal.
 4. Open http://127.0.0.1:5000 in your internet browser.
+
+
+
+### Run database container
+
+1. In terminal type in `docker-compose -f envs/dev/docker-compose.yml up --detach` command. (you need Docker and Docker Compose installed)
```

Comment:
Let's split this info into two sections. About Docker and Docker Compose requirements can be added to the bottom, where we have:
> __This project requires Python 3.10.__

And command we need to run the database should be in `Run application` section, because our application cannot work without database.